frontend/src/scenes/pipeline/ReplayMenu.tsx

```diff
@@ -0,0 +1,465 @@
+                loading={eventsLoading}
+                type="secondary"
+                icon={<IconRefresh />}
+                size="small"
+            >
+                Refresh
+            </LemonButton>
+            <DateFilter
+                dateFrom={baseEventsQuery?.after ?? undefined}
+                dateTo={baseEventsQuery?.before ?? undefined}
+                onChange={changeDateRange}
+            />
+        </div>
+    )
+}
+
+export function HogFunctionEventEstimates({ id }: { id: string }): JSX.Element | null {
+    const logic = hogFunctionReplayLogic({ id })
+    const { eventsLoading, eventsWithRetries, loadingRetries, totalEvents } = useValues(logic)
+    const { retryHogFunction } = useActions(logic)
+
+    return (
+        <LemonTable
+            dataSource={eventsWithRetries}
+            loading={eventsLoading}
+            loadingSkeletonRows={5}
+            pagination={{
+                controlled: true,
+                pageSize: eventsWithRetries.length,
+                hideOnSinglePage: false,
+                entryCount: totalEvents,
+            }}
+            expandable={{
+                noIndent: true,
+                expandedRowRender: ([, , retries]) => {
+                    return (
+                        <LemonTable
+                            dataSource={retries}
+                            embedded={true}
+                            columns={[
+                                {
+                                    title: 'Status',
+                                    key: 'status',
+                                    width: 0,
+                                    render: (_, retry) => {
+                                        return false ? (
+                                            <LemonBanner type={retry.status === 'success' ? 'success' : 'error'}>
+                                                {retry.status === 'success' ? 'Success' : 'Error'}
+                                            </LemonBanner>
+                                        ) : (
+                                            <RetryStatusIcon retries={[retry]} showLabel />
+                                        )
+                                    },
+                                },
+                                {
+                                    title: 'Test invocation logs',
+                                    key: 'testInvocationLogs',
+                                    render: (_, retry) => <RetryResults retry={retry} />,
+                                },
+                            ]}
+                        />
+                    )
+                },
+            }}
+            columns={[
+                {
+                    key: 'retries',
+                    title: 'Retries',
+                    width: 0,
+                    render: (_, [, , retries]) => {
+                        return <RetryStatusIcon retries={retries} />
+                    },
+                },
```

Comment:
I think we should remove this. It is confusing as it indicates a history of what happened when it is really only for this page.